docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -58,11 +58,13 @@ const theme = createTheme({
   components: {
     MuiButton: {
       styleOverrides: ({ ownerState }) => ({
-        ...(ownerState.variant === 'contained' &&
-          ownerState.color === 'primary' && {
-            backgroundColor: '#202020',
-            color: '#fff',
-          }),
+        root: ({ ownerState }) => ({
+          ...(ownerState.variant === 'contained' &&
+            ownerState.color === 'primary' && {
+              backgroundColor: '#202020',
+              color: '#fff',
+            }),
+        }),
       }),
```

Comment:
It should be like this:
```js
styleOverrides: {
  root: ({ ownerState }) => ({
    ...(ownerState.variant === 'contained' &&
      ownerState.color === 'primary' && {
        backgroundColor: '#202020',
        color: '#fff',
      }),
  }),
},
```

👍 Thanks for fixing it.

Reply:
Oops. Thanks for the correction!